libethereum/State.h

```diff
@@ -82,7 +82,8 @@ enum class BaseState
 enum class Permanence
 {
 	Reverted,
-	Committed
+	Committed,
+	Unfinished	///< uncommited state for changeLog readings
```

Comment:
In documenting comments always use full sentences. Must be `Uncommited state for change log readings in tests.".
Seeing the comment, shouldn't we change `Unfinished` to `Uncommited`?